feat(changelog): functions-changed-serverless-functions-logs-and 2023-06-13 (#1802)
scaleway/docs-content: changelog/june2023/2023-06-13-functions-changed-serverless-functions-logs-and.mdx

---

> title: Serverless Functions Logs and Metrics in Cockpit

Reviewer: ```suggestion
title: Serverless Functions logs and metrics are available in Cockpit!
```

status: changed
author:
    fullname: 'Join the #serverless-functions channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-06-13
category: serverless
product: functions
---

> Serverless Functions Logs and Metrics are now displayed in Cockpit by default

Reviewer 2: ```suggestion
Serverless Functions Logs and Metrics are now displayed in Cockpit by default, enabling more control over your logs and metrics.
```

> Serverless Functions Logs and Metrics are now displayed in Cockpit by default, enabling more control over your logs and metrics.

Reviewer: ```suggestion
Serverless Functions logs and metrics are now displayed in Cockpit by default, allowing you to have more control over your logs and metrics. Read [our documentation](/observability/cockpit/reference-content/cockpit-limitations/#product-integration-into-cockpit) to find out which other products are integrated into Cockpit.
```
